commcare_connect/program/tests/test_helpers.py

```diff
@@ -137,3 +141,122 @@ def test_failed_assessments(self):
         assert annotated_opp.workers_passing_assessment == 1
         assert annotated_opp.workers_starting_delivery == 2
         assert annotated_opp.percentage_conversion == 100.0
+
+
+@pytest.mark.django_db
+class TestDeliveryPerformanceReport:
```

Comment:
Can we create a Base Test Class to take out the common functionality here? That could be useful in adding more tests that require a Program Manager setup.

Reply:
Added [78fe8cd](https://github.com/dimagi/commcare-connect/pull/402/commits/78fe8cd983e6cb2381c1edd63f73eea1d9b4c2a3)